en/tutorials/linux_sbus.md

```diff
@@ -1,31 +1,33 @@
-# S.Bus Driver for Linux
+# Connecting an S.Bus Receiver on Linux
 
-The *S.Bus Driver for Linux* allows a Linux-based autopilot to access up to 16 channels from a *Futaba S.Bus receiver* via a serial port. The driver should also work with other receivers that use the S.Bus protocol, including as FrSky, RadioLink, and even S.Bus encoders. 
+This topic shows how to setup a linux-based autopilot to use an S.Bus reciever (or encoder - e.g. from Futaba, RadioLink, etc.) via any serial port.
```

Comment:
All other RC protocols (except PPM) should now work as well.

Reply:
Awesome. Do any of the other protocols need the inverter?

Reply:
I think non of them are inverted.

Reply:
@bkueng Thanks very much. Based on this I have *generalised* the topic to "Connecting an RC Receiver on Linux" and merged. Would appreciate a quick scan on the output: http://dev.px4.io/master/en/tutorials/linux_sbus.html

Reply:
All good.